fs/vfs/fs_epoll.c

```diff
@@ -365,8 +365,7 @@ int epoll_ctl(int epfd, int op, int fd, struct epoll_event *ev)
         return -1;
     }
 
-  poll_notify(&eph->poll, 1, eph->poll[0].events);
```

Comment:
Sorry for dummy question, but why this is changed?

Reply:
poll[0] initialize to `POLLIN | POLLFILE` here:
https://github.com/apache/incubator-nuttx/pull/7227/files#diff-47480c429464381a680229de81d4ed082bf0e07eca91d1b2bf8fd0c5aeaf4a64R198
so it's better to use POLLIN directly here to avoid pass the no event flag(POLLFILE) to poll_notify.

Reply:
So POLLOUT is not a use case here?

Reply:
Yes, it initialize to POLLIN and never change after that.